libbroker/broker/internal/core_actor.cc

```diff
@@ -132,7 +132,8 @@ core_actor_state::metrics_t::metrics_t(caf::actor_system& sys) {
   message_metric_sets[5].assign(proc.pong, buf.pong);
 }
 
-core_actor_state::core_actor_state(caf::event_based_actor* self,
+core_actor_state::core_actor_state(caf::event_based_actor* self, //
```

Comment:
We use comments like these in a couple places to influence `clang-format` behavior, i.e., to force a line break here and have it formatted consistently like this:

```c++
core_actor_state::core_actor_state(caf::event_based_actor* self, //
                                   prometheus_registry_ptr reg,
                                   endpoint_id this_peer,
                                   filter_type initial_filter,
                                   endpoint::clock* clock,
                                   const domain_options* adaptation,
                                   connector_ptr conn)
  : self(self),
```

Otherwise, `clang-format` will try to put things into fewer lines and break like this, even it makes things harder to read:

```c++
core_actor_state::core_actor_state(
  caf::event_based_actor* self, prometheus_registry_ptr reg,
  endpoint_id this_peer, filter_type initial_filter, endpoint::clock* clock,
  const domain_options* adaptation, connector_ptr conn)
  : self(self),
```

There are probably ways to consistently format it like the former by tweaking `clang-format` penalty parameters or maybe there is an actual setting to prefer consistent formatting over saving source lines. Didn't seem worth the time, though, since empty comments can fix the few places where `clang-format` makes weird decisions.